pandas/compat/numpy/__init__.py

```diff
@@ -1,21 +1,22 @@
 """ support numpy compatibility across versions """
 
-from distutils.version import LooseVersion
 import re
 
 import numpy as np
+from packaging.version import Version
+# from pandas.util.version import Version
```

Comment:
Use `from pandas.util.version import Version` here would cause import error.

```
pandas/__init__.py:22: in <module>
    from pandas.compat import (
pandas/compat/__init__.py:15: in <module>
    from pandas.compat.numpy import (
pandas/compat/numpy/__init__.py:7: in <module>
    from pandas.util.version import Version
pandas/util/__init__.py:1: in <module>
    from pandas.util._decorators import (  # noqa
pandas/util/_decorators.py:14: in <module>
    from pandas._libs.properties import cache_readonly  # noqa
pandas/_libs/__init__.py:13: in <module>
    from pandas._libs.interval import Interval
pandas/_libs/interval.pyx:1: in init pandas._libs.interval
    import numbers
pandas/_libs/hashtable.pyx:1: in init pandas._libs.hashtable
    cimport cython
pandas/_libs/missing.pyx:30: in init pandas._libs.missing
    from pandas.compat import IS64
E   ImportError: cannot import name IS64
```

Reply:
i'd like to see the pandas.compat import removed from _libs.missing anyway